tests/defaults.yml

```diff
@@ -8,3 +8,46 @@ parameters:
         keyC: valueC
       wouldYou:
         likeToC: valueC
+    secrets:
+      tlsA:
+        metadata:
+          name: tlsA
```

Comment:
Note that you don't need to specify `metadata.name`, since that's inferred from the key in `secrets` if you use `com.generateResources()`.

Reply:
Removed